starknet_py/net/signer/base_signer.py

```diff
@@ -1,7 +1,7 @@
 from abc import ABC, abstractmethod
 from typing import List
 
-from starknet_py.net.models.transaction import Transaction
+from starknet_py.net.models import Transaction
```

Comment:
I'd consider getting rid of imports from `models.__init__.py` completely. Things like this can cause circular import problems.

Reply:
https://github.com/software-mansion/starknet.py/issues/475